src/workers/notifications/squadPostModerationApproved.ts

```diff
@@ -0,0 +1,28 @@
+import { generateTypedNotificationWorker } from './worker';
+import { NotificationType } from '../../notifications/common';
+import { NotificationPostContext } from '../../notifications';
+import { buildPostContext } from './utils';
+
+const worker =
```

Comment:
Just naming wise again, should we name `source` instead of squad, so its generic and squads are just sources under the hood

Reply:
Yeah, we can refactor to that to keep it consistent.